postman/Get Broadcast Stats.postman_collection.json

```diff
@@ -0,0 +1,570 @@
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists on Account",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "4d22c8fa-8695-4098-a275-10658e8aaea5",
+						"exec": [
+							"var jsonData = pm.response.json();",
+							"",
+							"pm.test(\"Successfully selected a random listId\", function() {",
+							"    var list_entry = Math.floor((Math.random() * jsonData.entries.length ));",
+							"    pm.globals.set(\"listId\",jsonData.entries[list_entry].id );",
+							"});",
+							"",
+							"pm.test(\"Successfully submitted request\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Random List",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "b41770d3-1044-48f5-9536-6cf9ba89088a",
+						"exec": [
+							"var jsonData = pm.response.json();",
+							"pm.globals.set(\"sent_broadcasts_link\", '');",
+							"",
+							"pm.test(\"Successfully retrieved sent_broadcast_link from random list\", function() {",
+							"    pm.globals.set(\"sent_broadcasts_link\", jsonData.sent_broadcasts_link);",
+							"});",
+							"",
+							"pm.test(\"Successfully submitted request\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Total Sent Broadcasts",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "e238f70c-3010-41a2-ba60-6fd4e312850d",
+						"exec": [
+							"var jsonData = pm.response.json();",
+							"pm.globals.set(\"broadcast_self_link\", '');",
+							"",
+							"pm.test(\"Successfully retreived sent broadcasts from list\", function() {",
+							"    if (pm.expect(jsonData.entries.length).to.be.above(0)) {",
```

Comment:
You don't need to wrap the expect in an if statement. It will stop the test if it fails.

Reply:
Updated